Automattic/wp-calypso
# Jetpack Checklist Modal: Migrate Fireworks Image

#### Changes proposed in this Pull Request

This migrates the one image usage of the fireworks illustration to Webpack. For details on the reasoning behind this, see #36850.

#### Testing instructions

There should only be one use of this image: https://github.com/Automattic/wp-calypso/search?q=%22fireworks.svg%22&unscoped_q=%22fireworks.svg%22

You can find it by connecting a Jetpack site to WordPress.com, where the "Thank you" modal should flash. Otherwise, it may be necessary to actually make a purchase. Just verify that the image is exactly the same as before. :) (cc @sirreal, @ockham)

<img width="1296" alt="Screenshot 2019-12-11 at 20 49 29" src="https://user-images.githubusercontent.com/43215253/70659490-18faa280-1c58-11ea-9bf2-af51e65fe18a.png">
